### DISCUSSION TITLE
Gem group channels

### DISCUSSION THREAD

**Discussion by ilariima** [iptv-org/community#1072] [Q&A]
Hey people, I'm trying to find Gem group channels but it seems they are not here.
They have a bunch of channels that are very important for the Persian community.

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
